components/Timeline.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @next/next/no-img-element */
+import { TIMELINE_DATA } from "../data/timeline";
+
+const TIMELINE_CONTAINER = "bg-white flex flex-col";
+
+const Timeline = () => {
+  const middleLine = <div className="border-l-4 border-[#7055FD] h-30" />;
+
+  const renderTimeline = TIMELINE_DATA.map(({ year, description, smallImage, largeImage }, index) => {
+    if (index % 2 == 0) {
+      return (
+        <div className="flex flex-row flex-start">
```

Comment:
Might be more readable if we breakout these into `LeftSide` and `RightSide` components internal to this file (do not export)

Reply:
Good point